owncloud/core
# Collaborative tags new category work

## Description

Adding a new tag which is same as restricted tag except it is not editable by users.

## Related Issue




- Fixes <issue_link>

## Motivation and Context

Adding a new tag which is same as restricted tag except it is not editable by users.

## How Has This Been Tested?



- Create a new tag `Editable` with name lets say , `editableTag`, by enabling systemtag_management app by navigating to admin page -> `Workflow & tags`
- Create groups `group1`, `group2` under users page ( by enabling user_management app)
- Create users `user1`, `user2`. Add them to `group1`
- Create user `user3` and add it to `group2`.
- Now try to update the `editableTag` in the `Workflow & tags` page, by adding 2 groups `group1` and `group2`.
- Login to `user1`, now the user will see that `editableTag` is editable. It can also be assigned/unassigned.
- Login to `user3`, now the user will see that `editableTag` is not editable. Its only assignable/unassignable.
- Similarly create `visibleTag` under `Visible`, `invisibleTag` under `Invisible`, and `restrictedTag` under `Restricted`. The `editableTag` behaves the same way as it was before.
- Add `group1` to `restrictedTag`.
- Now when logged in as `user3` , `restrictedTag` is not visible. Only `visibleTag` and `editableTag` are visible.


## Screenshots (if appropriate):

## Types of changes


## Checklist:


- [ ] Requires: https://github.com/owncloud/systemtags_management/pull/30

## Open tasks:
